client/src/Components/Pages/Search/index.js

```diff
@@ -0,0 +1,220 @@
+import React, { Component } from "react";
+import axios from "axios";
+import Autosuggest from "react-autosuggest";
+
+// UI helper functions
+import {
+  SVGCreator,
+  StarRateCreator,
+  SortArrayNewest
+} from "../../../UIHelper";
+
+// styles
+import {
+  HeadlineDiv,
+  RowDiv,
+  ItemDiv,
+  SearchLegendDiv,
+  SymbolDiv,
+  OrganisationDetailsDiv,
+  ReviewDetailsDiv,
+  InnerDivLastReviews,
+  InnerDivSuggestions,
+  ArrowDiv,
+  SearchWrapper,
+  SuggestionBox,
+  AddItemBox,
+  LegendTitle,
+  ReviewsFrame,
+  ProfileLink
+} from "./Search.style";
+
+import { organizationIcons } from "./../../../theme";
+
+export default class Search extends Component {
+  state = {
+    loaded: false,
+    data: null,
+    value: "",
+    suggestions: []
+  };
+
+  componentDidMount() {
+    axios.get("/api/search").then(organizations => {
+      this.setState({
+        data: organizations.data,
+        loaded: true
+      });
+    });
+  }
+
+  // functions for autosuggest component
+  // create suggestions array based on user input
+  getSuggestions = (value, organisationsArray) => {
+    const inputValue = value.trim().toLowerCase();
+    const inputLength = inputValue.length;
+    const suggestions = organisationsArray.filter(
+      orga => orga.name.toLowerCase().slice(0, inputLength) === inputValue
+    );
+
+    // in case there are no suggestions still return a value enabling the 'add' box to be rendered
+    if (suggestions.length === 0) {
+      return [{ isEmpty: true }];
+    }
+    return suggestions;
+  };
+
+  // gets called when suggestions gets clicked
+  getSuggestionValue = suggestion => suggestion.name;
+
+  // Autosuggest will call this function every time suggestions are updated
+  onSuggestionsFetchRequested = ({ value }) => {
+    this.setState({
+      suggestions: this.getSuggestions(value, this.state.data)
+    });
+  };
+
+  // Autosuggest will call this function every time you need to clear suggestions.
+  onSuggestionsClearRequested = () => {
+    this.setState({ suggestions: [] });
+  };
+
+  onChange = (event, { newValue }) => {
+    this.setState({ value: newValue });
+  };
+
+  // render functions
+
+  // renders individual suggestions in autosuggest search section
+  renderSuggestion = suggestion => {
+    // check if no suggestion available and return so that renderSuggestionsContainer function is still being called (gets deactivated otherwise)
+    if (suggestion.isEmpty) {
+      return;
```

Comment:
`return null`

returning `null` is better than returning `undefined`. When you're explicitly returning nothing its good to use `null` so that you know that its been written by you (since the code will never produce `null` unless you write it, but it can produce `undefined`s)